manifests/action.pp

```diff
@@ -0,0 +1,31 @@
+# == Define: fail2ban::action
+#
+define fail2ban::action (
+  String $actionban = undef,
+  String $actionunban = undef,
+
+  Optional[String] $config_file_owner = $fail2ban::config_file_owner,
```

Comment:
can you please align all the `=` chars and enforce a minimal string length of 1?
```suggestion
  Optional[String[1]] $config_file_owner = $fail2ban::config_file_owner,
```